openslides_backend/action/actions/committee/update.py

```diff
@@ -73,9 +172,44 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
                     "manager_ids",
                 ]
             ]
-        ):
+        ) or ("parent_id" in instance and not instance["parent_id"]):
             raise MissingPermission(OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION)
 
+        if "parent_id" in instance:
+            data = self.datastore.get_many(
+                [
+                    GetManyRequest(
+                        "committee",
+                        [instance["parent_id"], instance["id"]],
+                        ["all_parent_ids"],
+                    ),
+                    GetManyRequest(
+                        "user", [self.user_id], ["committee_management_ids"]
+                    ),
+                ],
+                lock_result=False,
+            )
+            committee_ancestors = [
+                com.get("all_parent_ids", []) for com in data["committee"].values()
+            ]
+            parent_intersection = set(committee_ancestors[0]).intersection(
+                committee_ancestors[1]
+            )
```

Comment:
I'm not sure if this is actually more readable. You are free to disregard.
```suggestion
        if "parent_id" in instance:
            parent_id = instance["parent_id"]
            id_ = instance["id"]
            both_committees = self.datastore.get_many(
                [
                    GetManyRequest(
                        "committee",
                        [parent_id, id_],
                        ["all_parent_ids"],
                    ),
                    GetManyRequest(
                        "user", [self.user_id], ["committee_management_ids"]
                    ),
                ],
                lock_result=False,
            )["committee"]
            parent_intersection = set(both_committees[parent_id].get("all_parent_ids", [])).intersection(
                 set(both_committees[id_].get("all_parent_ids", []))
            )
```

Reply:
Unfortunately this won't work, because the user data can't be loaded then, if I account for that, my code ends up being more readable.